code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -192,7 +193,7 @@
 
 	else //Just siphoning all air
 
-		var/transfer_moles = environment.total_moles() * (volume_rate / environment.volume)
+		var/transfer_moles = total_moles * (volume_rate / environment.volume)
```

Comment:
is there a point in caching it here? since these are in two different if blocks

Reply:
Now that I think about it, no.